### ISSUE TITLE
FA Description field not mapping.

### ISSUE THREAD

**Issue by kelea99** [pulibrary/figgy#4137]
Some components are not showing the metadata associated with it from the finding aid. there does not seem to be a pattern to this. Example:
FA:  https://findingaids.princeton.edu/collections/WC064/c5518
Figgy: https://figgy.princeton.edu/catalog/8f6c3c4c-770e-4915-9968-15ae81965f32
DPUL: https://dpul.princeton.edu/catalog/2n49t210z

Example of item where metadata is showing up in Figgy, FA, DPUL:
FA: https://findingaids.princeton.edu/collections/WC064/c4064
Figgy: https://figgy.princeton.edu/catalog/0c1506aa-8d46-423d-a76f-d7bad96da172
DPUL:https://dpul.princeton.edu/pudl0017/catalog/v405s975c

**Comment by tpendragon** [comment]
It looks like these items just need their metadata reimported. A bunch of titles are wrong too, maybe we should just update metadata for everything.

**Comment by kelea99** [comment]
@tpendragon probably good to update the lot of it. @cwulfman any reason why this would be a terrible idea? I wouldn't think so, as all Metadata for WA is coming from FA.

**Comment by tpendragon** [comment]
I'm going to update the metadata for everything in WC064, since we don't have the bandwidth to do everything right now. I'll ping you to check on it when it's done?

**Comment by kelea99** [comment]
Perfect. will do.

**Comment by tpendragon** [comment]
@kelea99 Can you take a look when you have a chance? Should be done.

**Comment by kelea99** [comment]
@tpendragon. The collection metadata still isnt mapping completely for all items the collection.  

Example:
Figgy: https://figgy.princeton.edu/catalog/0009454e-8786-4be6-a343-1258452ec5a1
FA: https://findingaids.princeton.edu/collections/WC064/c4690

**Comment by tpendragon** [comment]
@kelea99 I just accidentally fixed that one. Can you find another example? It looks like refreshing through the UI is working but the job isn't, for some reason.

**Comment by kelea99** [comment]
yup, @tpendragon ! Example 02 (also about to look for any possible similarities to the problem items): 
FA: https://findingaids.princeton.edu/collections/WC064/c4675
Figgy: https://figgy.princeton.edu/catalog/22600dc1-3d05-423b-9968-dd5cf7900f86

**Comment by tpendragon** [comment]
Yeah I figured out the issue. It's updating all the metadata again, I'll ping you when it's ready for another look.

**Comment by kelea99** [comment]
problems seem to be mostly (if not entirely) from WC064 collection. WC054 and WC055 seem in tact, after intitating a selective search

**Comment by tpendragon** [comment]
@kelea99 Can you take another look? They should be done, but we had some FA issues last night.

**Comment by kelea99** [comment]
@tpendragon, everything looks good except for the fact that the now the FA description field is no longer being mapped for any of the FA materials in WC064,WC055, WC054.

**Comment by kelea99** [comment]
See related ticket:  https://github.com/pulibrary/figgy/issues/3914
